.github/workflows/compile_v_with_vtcc.sh

```diff
@@ -14,23 +14,27 @@ show "Clone vtcc"
 du -s vtcc/
 ## TODO: just `./v vtcc`, later will cause V, to detect the compiler as tcc (which it is), and add `-fwrapv`, which causes the vtcc compiler to panic currently
 show "Compile vtcc"
-./v -o vtcc/xx vtcc/
-ls -la vtcc/xx
-./vtcc/xx --version
+#./v -o vtcc/xx vtcc/
+cd vtcc/
+v run make.vsh
+cd ..
+
+ls -la vtcc/vtcc
+./vtcc/vtcc --version
 
 show "Generate the C file, for the current V version"
 ./v -o vlang.c cmd/v
 ls -la vlang.c
 
 show "Compile the C file with vtcc"
-export tcclib=thirdparty/tcc/lib/tcc
-export tccinc=$tcclib/include
-./vtcc/xx -o v_compiled_with_vtcc vlang.c -L$tcclib -I$tccinc -lc -ldl -pthread -ltcc1 $tcclib/bt-log.o
```

Comment:
oh, it works now without the flags?
nice

Reply:
Yep :-)